sources/iTermAdvancedSettingsModel.m

```diff
@@ -65,7 +65,6 @@ + (NSString *)name { \
 
 #pragma mark Terminal
 DEFINE_BOOL(traditionalVisualBell, NO, @"Terminal: Visual bell flashes the whole screen, not just a bell icon.");
-DEFINE_FLOAT(antiIdleTimerPeriod, 60, @"Terminal: Anti-idle interval in seconds.\nWill not go faster than 60 seconds.");
```

Comment:
A migration path should be provided for users who had this set by using a computed default value.